### ISSUE TITLE
viennarna 2.1.8 - fatal error: gd.h: No such file or directory

### ISSUE THREAD

**Issue by tseemann** [Homebrew/homebrew-science#1965]
I think this formula doesn't have a dependency on the GD library _from Homebrew_. 

It probably works for most people becuase it is falliing back to a system gd-devel package installed in /usr. I know I don't have that installed.

I *do* have ````/bio/linuxbrew/include/gd.h``` but the ```./configure``` isn't using it.

```
./configure
--prefix=/bio/linuxbrew/Cellar/viennarna/2.1.8
--disable-debug
--disable-dependency-tracking
--disable-openmp
--without-perl

<snip>

/usr/bin/gcc  -I./src -Os -w -pipe -march=core2  -I/usr/local/include -DPACKAGE_NAME=\"\" -DPACKAGE_TARNAME=\"\" -DPACKAGE_VERSION=\"\" -DPACKAGE_STRING=\"\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DLINUX=1 -DDO_PS=1 -DDO_FIG=1 -DDO_X11=1 -DDO_GD=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_LIMITS_H=1 -c src/GD/g2_gd.c -o src/GD/g2_gd.o
In file included from src/GD/g2_gd.c:29:0:
src/GD/g2_gd_P.h:22:16: fatal error: gd.h: No such file or directory
 #include <gd.h>
                ^
compilation terminated.
make[4]: *** [src/GD/g2_gd.o] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: Leaving directory `/tmp/viennarna-dL7APL/ViennaRNA-2.1.8/RNAforester/g2-0.70'
make[3]: *** [all-recursive] Error 1
make[3]: Leaving directory `/tmp/viennarna-dL7APL/ViennaRNA-2.1.8/RNAforester'
make[2]: *** [all] Error 2
make[2]: Leaving directory `/tmp/viennarna-dL7APL/ViennaRNA-2.1.8/RNAforester'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/tmp/viennarna-dL7APL/ViennaRNA-2.1.8'
make: *** [all] Error 2
```

**Comment by saketkc** [comment]
Does this help?
```
$./configure CPPFLAGS="-I /bio/linuxbrew/include/"

```

**Comment by tseemann** [comment]
Can't try right now but I assume it will need a ```LDFLAGS=-L/bio/linuxbrew/lib``` too?

**Comment by iml** [comment]
Should be fixed by 487391941503607ad679d1d2e1784b69ce53675e.
